blob/gcsblob/gcsblob_integration_test.go

```diff
@@ -24,11 +24,12 @@ import (
 	"github.com/google/go-cloud/gcp"
 )
 
+var record = flag.Bool("record", false, "whether to record")
```

Comment:
Group these two together and the description should be more clear.

Reply:
Done.